content/applications/sales/crm/acquire_leads/convert.rst

```diff
@@ -2,66 +2,76 @@
 
-.. image:: convert/convert01.png
+.. image:: convert/convert-leads-leads-setting.png
     :align: center
+    :alt: Leads setting on CRM configuration page.
 
-You will now have a new submenu *Leads* under *Leads* where they
-will aggregate.
+This adds a new menu, :guilabel:`Leads`, to the menu bar at the top of the screen.
 
-.. image:: convert/convert02.png
+.. image:: convert/convert-leads-leads-menu.png
     :align: center
+    :alt: Leads menu on CRM application.
+
+Once the *leads* setting has been activated, it will apply to all sales teams by default. To turn
+off leads for a specific team, navigate to :menuselection:`Configuration --> Sales Team`. Select a
+team from the list to open the record, then uncheck the :guilabel:`Leads` box. Click
+:guilabel:`Save`.
+
+   .. image:: convert/convert-leads-leads-button.png
+       :align: center
+       :alt: Leads menu on CRM application.
 
 Convert a lead into an opportunity
 ==================================
 
-When you click on a *Lead* you will have the option to convert it to
-an opportunity and decide if it should still be assigned to the same
-channel/person and if you need to create a new customer.
+To convert a lead into an *opportunity*,
+When you click on a *Lead* you will have the option to convert it to an opportunity and decide if it
+should still be assigned to the same channel/person and if you need to create a new customer.
 
-.. image:: convert/convert03.png
+.. image:: convert/convert-leads-convert-opp-button.png
     :align: center
+    :alt: Create opportunity button.
 
-If you already have an opportunity with that customer Odoo will
-automatically offer you to merge with that opportunity. In the same
-manner, Odoo will automatically offer you to link to an existing
-customer if that customer already exists.
+.. image:: convert/convert-leads-conversion-action.png
+    :align: center
+    :alt: Create opportunity pop-up.
 
-Merge opportunities
-===================
+If you already have an opportunity with that customer Odoo will automatically offer you to merge
+with that opportunity. In the same manner, Odoo will automatically offer you to link to an existing
+customer if that customer already exists.
 
-Odoo will also automatically propose to merge opportunities if they have
-the same email address. When merging opportunities, Odoo merges the
-information into the opportunity which was created first, giving
-priority to the information present on the first opportunity.
+Merge leads and opportunities
+=============================
 
-No information is lost: data from the other opportunity is logged in the
-chatter and the information fields for easy access.
+Odoo will also automatically propose to merge opportunities if they have the same email address.
+When merging opportunities, Odoo merges the information into the opportunity which was created
+first, giving priority to the information present on the first opportunity.
```

Comment:
```suggestion
Odoo also automatically proposes to merge opportunities if they share the same email address, with a
:guilabel:`Similar Lead` smart button, located on the lead form.

When merging opportunities, Odoo merges the information into the first created opportunity, giving
priority to the information present on the first opportunity.
```

Reply:
It might be nice to further explain what is seen/occurs when that smart button is clicked -- just to flush things out a touch.